### PR TITLE
AT_01.06.05_ | Header My views menu link

### PR DESCRIPTION
**Pull request by SvetlanaKazakova**
https://trello.com/c/Ol13jogw/896-at010605-header-my-views-menu-link

### CODE REVIEW

**Review by puh50 — COMMENTED** [review]
Need to divide the case on several cases to check transferring to correct page, existing left menu items

**Review by SvetlanaKazakova — COMMENTED** [review]
ok

**Comment by Paul-Korz** [comment]
Hello, this test work only for you. And for other users who have username - admin. You have a problem with css selectors

File: cypress/e2e/headerMyViewMenuLinks.cy.js [hunk 1]
```diff
@@ -9,4 +9,12 @@ describe("US_01.06 Header My views menu link", () => {
         cy.get('@sidePanel').find('[href="/asynchPeople/"]').should('be.visible');
         cy.get('@sidePanel').find('[href$="builds"]').should('be.visible');
     })
+    it('TC_01.06.05 Header My views menu link',()=>{
+        cy.get('a[href="/user/admin"] .jenkins-menu-dropdown-chevron').click({force: true});
+        cy.get('[href="/user/admin/my-views"]').click();
+        cy.get('.content-block').should('have.text','Create a job');
+        cy.get('[href="/user/admin/my-views/view/all/newJob"]').should('have.text','New Item');
+        cy.get('[href="/asynchPeople/"] .task-link-text').should('have.text','People');
+        cy.get('[href="/user/admin/my-views/view/all/builds"]').should('have.text','Build History');
```
Comment (Rom-T) [inline cypress/e2e/headerMyViewMenuLinks.cy.js:18]: Please move `data` in the fixtures

**Pull request merged**